repo/packages/B/bookkeeper/0/package.json

```diff
@@ -0,0 +1,20 @@
+{
+ "packagingVersion": "3.0",
+ "name": "bookkeeper",
+ "version": "4.3.4",
+ "scm": "https://github.com/twitter/bookkeeper",
+ "maintainer": "jia.zhai@emc.com",
+ "description": "bookkeeper",
```

Comment:
Please add a better description. 
From the `example` you submitted: 
BookKeeper is a replicated log service which can be used to build replicated state machines. A log contains a sequence of events which can be applied to a state machine. BookKeeper guarantees that each replica state machine will see all the same entries, in the same order. One bookkeeper instance is called a bookie, By default this package deploy 1 bookie. Usually we should deploy at least 3 bookies to start a bookkeeper cluster.